src/components/MainContent.js

```diff
@@ -1,21 +1,39 @@
-import { 
-  BsFillPersonFill, FiMenu, FcSettings, IoNotificationsSharp, FaCouch, BsPersonFill
-} from '../utils/icons';
-import CompanyInfo from './CompanyInfo';
+import {
+  BsFillPersonFill,
+  FiMenu,
+  FcSettings,
+  IoNotificationsSharp,
+  FaCouch,
+  BsPersonFill,
+  ImCheckmark,
+  HiDotsVertical,
+} from "../utils/icons";
+import CompanyInfo from "./CompanyInfo";
+import "./MainContent.css";
 
 export default function MainContent() {
   return (
-    <div className="w-screen h-screen grid grid-cols-2 border-[2px] border-white overflow-y-auto">
-      <div className="w-[calc(100vw-238px)] h-full ml-[218px] grid grid-cols-6 grid-rows-12 border-[2px] border-black">
+    <div className="box w-screen h-screen grid grid-cols-2 overflow-y-auto bg-gray-200">
+      <div className="w-[calc(100vw-238px)] h-full ml-[218px] grid grid-cols-6 auto-rows-min gap-5">
         {/* Nav section */}
```

Comment:
Whenever you need comments to organize your code, there's a chance it might be a signal that refactoring is what's actually needed.

This Nav section can become a component of its own `<nav>` for example